File: app/src/main/java/com/ivantha/playtolearn/activity/ProfileActivity.java
package com.ivantha.playtolearn.activity;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

import com.ivantha.playtolearn.R;

public class ProfileActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_profile);
    }

    @Override
    protected void onActivityResult(int requestCode, int resultCode, Intent data) {
        super.onActivityResult(requestCode, resultCode, data);
    }

    public void onClickLoadButton(View view) {
    }
}

File: app/src/main/java/com/ivantha/playtolearn/activity/ProfileActivity.kt
package com.ivantha.playtolearn.activity

import android.os.Bundle
import android.support.v7.app.AppCompatActivity

import com.ivantha.playtolearn.R
import com.ivantha.playtolearn.common.Session
import kotlinx.android.synthetic.main.activity_profile.*

class ProfileActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_profile)

        // Show Current Level
        var level: Int = Session.saveFile!!.currentLevel.id
        levelTextView.text = level.toString()

        // Show Total Score
        // TODO: Get total score

        // Show Current Rank
        // TODO: Get current rank
    }


}
